src/client/src/tscParseLineProtocol.c

```diff
@@ -2311,10 +2301,11 @@ static int32_t parseSmlValue(TAOS_SML_KV *pKV, const char **idx,
             ret = TSDB_CODE_TSC_LINE_SYNTAX_ERROR;
             goto error;
           }
-
+          if (*cur == '"') {
```

Comment:
For SML binary str encoding, the surrounding double-quotation marks used as identifying characteristics of its type will be cut off directly when being converted to type `TSDB_DATA_TYPE_BINARY` later in fn `convertSmlValueType`. So, the quotation marks within the string need not be  escaped with a backslash '\\' this time. Thus, the preceding backslash of an internal quotation mark is skipped and then stripped here.